core/packages/test/config/launch-config.toml

```diff
@@ -43,23 +48,23 @@ chain_spec_path = "{{output_dir}}/spec.json"
 cumulus_based = true
 id = 1000
 
-[[parachains.collators]]
-command = "{{output_bin_dir}}/snowbridge"
-name = "snowbrige-alice"                              
-rpc_port = 8081
-validator = true
-ws_port = 11144
-args = ["--enable-offchain-indexing=true","--pruning=archive"]
+    [[parachains.collators]]
+    command = "{{output_bin_dir}}/snowbridge"
+    name = "snowbrige-alice"
+    rpc_port = 8081
+    validator = true
+    ws_port = 11144
+    args = ["--enable-offchain-indexing=true","--pruning=archive"]
 
-[[parachains.collators.env]]
-name = "RUST_LOG"
-value = "runtime=debug,parachain=trace,cumulus-collator=trace,aura=trace,xcm=trace"
+        [[parachains.collators.env]]
+        name = "RUST_LOG"
+        value = "runtime=debug,parachain=trace,cumulus-collator=trace,aura=trace,xcm=trace"
```

Comment:
better to add `ethereum-beacon-client` to log

Reply:
I've gone with trace logs here, since we have some trace logs in this pallet and no debug logs